articles/azure-functions/functions-infrastructure-as-code.md

```diff
@@ -215,7 +219,8 @@ A function app has many child resources that you can use in your deployment, inc
         "alwaysOn": true,
         "appSettings": [
             { "name": "FUNCTIONS_EXTENSION_VERSION", "value": "~1" },
-            { "name": "Project", "value": "src" }
+            { "name": "Project", "value": "src" },
+            { "name": "WEBSITE_NODE_DEFAULT_VERSION", "value": "6.5.0" }
```

Comment:
Why would you move the Node version from the Site Config to the App Settings? Both should land the value into the App Settings, is there a reason why you want the version to be set after the site has been created instead of with the creation of the site?

Reply:
my first time working with ARM template, but site/config declared site as a dependency. I thought that means site has to be created before site/config, which is the opposite of what you were saying? Anyway I was following the default ARM template used when creating azure function in the portal